its/src/test/java/com/sonar/it/csharp/SdkTest.java

```diff
@@ -39,16 +39,36 @@ public class SdkTest {
   @TempDir
   private static Path temp;
 
+  @Test
```

Comment:
File `net6.sln` is called `net6.sln` instead of `Net6.sln` like everything else :)